include/picongpu/particles/atomicPhysics/stage/RollForOverSubscription.hpp

```diff
@@ -68,10 +73,12 @@ namespace picongpu::particles::atomicPhysics::stage
 
             auto& ions = *dc.get<IonSpecies>(IonSpecies::FrameType::getName());
 
-            using RejectionProbabilityCacheField
-                = localHelperFields::RejectionProbabilityCacheField<picongpu::MappingDesc>;
-            auto& rejectionProbabilityCacheField
-                = *dc.get<RejectionProbabilityCacheField>("RejectionProbabilityCacheField");
+            auto& rejectionProbabilityCacheField_Bin
```

Comment:
note the clean and save way is:

```C++
auto rejectionProbabilityCacheField_Bin
                = dc.get<localHelperFields::RejectionProbabilityCacheField_Bin<picongpu::MappingDesc>>(
                    "RejectionProbabilityCacheField_Bin");
```

and later 

```C++
rejectionProbabilityCacheField_Bin->getDeviceDataBox(),
```

keeping the object as reference is dangerous can is an easy way to get segfaults

Reply:
fixed